identity/app/views/profileForms.scala.html

```diff
@@ -79,7 +86,9 @@ <h1 class="identity-title" data-test-id="edit-profile-header">Edit your profile<
                 @content(5, "/email-prefs")(profile.privacyForm(idUrlBuilder, idRequest, user, forms.privacyForm, emailPrefsForm, emailSubscriptions, availableLists)))
             }
         </div>
-    </div>
+    </div></div>
```

Comment:
One too many closing tags here?

Reply:
WORSE this file is indentation hell right now bc i wanted it to be a light commit line-wise at the time, gonna sort it out